server/hangouts/methods.js

```diff
@@ -3,7 +3,9 @@ Meteor.methods({
     check(data, Match.ObjectIncluding({
       topic: String,
       slug: String,
-      description: String,
+      description: Match.ObjectIncluding({
```

Comment:
I'm not totally sure about this `Math.Any`.

The reason for this change is that description will now be submitted in this format https://quilljs.com/guides/working-with-deltas/